### ISSUE TITLE
[Test] Log statements 

### ISSUE THREAD

**Issue by turbolent** [onflow/cadence-tools#103]
### Issue To Be Solved

Currently, `log` statements are ignored when running tests. 

### Suggest A Solution

It would be great to get all log statements to get printed with some flag when running tests.

### Context

https://github.com/orgs/onflow/discussions/1330

**Comment by turbolent** [comment]
@m-Peter Would this maybe be part of the developer grant?

**Comment by m-Peter** [comment]
@turbolent Yes, indeed. That functionality is roughly described in this [issue](https://github.com/onflow/cadence/issues/1889), under the `Logs & Events` section.
